packages/webhooks/src/commands/webhooks/deliveries/index.ts

```diff
@@ -0,0 +1,67 @@
+import {flags} from '@heroku-cli/command'
+import {cli} from 'cli-ux'
+
+import BaseCommand from '../../base'
+
+export default class Deliveries extends BaseCommand {
+  static description = 'list webhook deliveries on an app'
+
+  static examples = [
+    '$ heroku webhooks:deliveries'
+  ]
+
+  static flags = {
+    app: flags.app(),
+    remote: flags.remote(),
+    status: flags.string({char: 's', description: 'filter deliveries by status'}),
+    pipeline: flags.pipeline({char: 'p', description: 'pipeline on which to list', hidden: true})
+  }
+
+  async run() {
+    const {flags} = this.parse(Deliveries)
+    let {path, display} = this.webhookType(flags)
+    const max = 1000
+
+    path = `${path}/webhook-deliveries`
+    if (flags.status) {
+      path += `?eq[status]=${encodeURIComponent(flags.status)}`
+    }
+
+    const {body} = await this.httpClient.get(path, {
```

Comment:
I think `httpClient` is misleading, it should be named `webhooksClient` or something like that.

Reply:
This should be `const {body: deliveries} = ...`

Reply:
This doesn't have a return type `this.httpClient.get<SomeTypeFoo>`, is that b/c it is missing in heroku-api-schema?